app/src/features/authentication/signup/schema/signup.ts

```diff
@@ -0,0 +1,19 @@
+import * as yup from "yup";
+
+export const schema = yup.object({
+  username: yup.string().required("Please enter a username"),
+  email: yup
+    .string()
+    .email("Please enter a valid email")
+    .required("Please enter an email"),
```

Comment:
I think me & you have to use the same messages for the required validation (which one will we use XD)

Reply:
yours are less nice to users...we'll use yours XD

Reply:
you mean we'll use yours..i just want it to be "your email" instead of "an email" XD